torch/fx/proxy.py

```diff
@@ -150,6 +150,10 @@ def no_node(arg):
         elif isinstance(a, base_types) or a is None or a is ...:
             return a
 
+        # higher-order operator
+        if isinstance(a, torch.fx.graph_module.GraphModule):
+            return a
+
```

Comment:
Can we get the prototype to work without adding these lines?

When I wrote this originally, if these three lines weren't here, then FX would complain that functions cannot accept GraphModules as inputs.

Reply:
Yea, we don't need this, rm.